mir_eval/pattern.py

```diff
@@ -303,10 +297,13 @@ def establishment_FPR(reference_patterns, estimated_patterns,
     return f_measure, precision, recall
 
 
-def occurrence_FPR(reference_patterns, estimated_patterns, thres=.75,
-                   similarity_metric="cardinality_score"):
-    """Establishment F1 Score, Precision and Recall.
-
+def occurrence_FPR(
+    reference_patterns,
+    estimated_patterns,
+    thres=0.75,
+    similarity_metric="cardinality_score",
+):
+    """Compute the occurrence F1 Score, Precision and Recall.
```

Comment:
@urinieto tagging you on this - there seemed to be some copypasta in this metric before, where it referred to computing establishment metrics though the function here is doing occurrence.  I've corrected it, I think, but I'd like to confirm with you.

Reply:
Oh, totally copypasta, nice catch! Lgtm